src/core/Microsoft.PowerToys.Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -1,134 +1,130 @@
-﻿<Page
```

Comment:
Next time, please do just functional changes and keep style changes for a separate PR. It is really hard to follow what are real changes and what is just formatting

Reply:
I can split this PR into two separate PRs when you need ^^

Reply:
Lets keep it for now